tools/ARIAtools/productPlot.py

```diff
@@ -569,9 +568,9 @@ def main(inps=None):
     # pass number of threads for gdal multiprocessing computation
     if inps.num_threads.lower()=='all':
         import multiprocessing
-        print('User specified use of all %s threads for gdal multiprocessing'%(str(multiprocessing.cpu_count())))
+        log.info('User specified use of all %s threads for gdal multiprocessing %s', multiprocessing.cpu_count())
```

Comment:
Codacy found an issue: [Not enough arguments for logging format string](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=6018914)